modin/experimental/engines/omnisci_on_ray/frame/data.py

```diff
@@ -451,6 +454,32 @@ def fillna(
 
         return new_frame
 
+    def dropna(self, subset, how="any"):
+        how_to_merge = {
+            "any": "AND",
+            "all": "OR",
+        }
+
+        # If index columns are not presented in the frame, then we have to create them
+        # based on "rowid". This is needed because 'dropna' preserves index.
+        if self._index_cols is None:
+            base = self._materialize_rowid()
+
+        checks = [base.ref(col).is_not_null() for col in subset]
```

Comment:
Isn't it possible a case when `base` is `None`? I see `base` is not `None` in case `self._index_cols` is `None`.

Reply:
you're right, going to fix it.